webapp/context.py

```diff
@@ -39,16 +39,6 @@ def releases():
         return yaml.load(releases, Loader=yaml.FullLoader)
 
 
```

Comment:
Remove this blank line here and linting should pass

Reply:
Lol great minds think alike @ndv99, I was doing just that while you left this comment 🙂. Running `dotrun format-python` usually takes care of this as well